app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post, foreign_key: 'author_id'
+
```

Comment:
Please add a method in the Comment model that updates the comments counter for a post.